extensions/wikia/AdEngine/js/video/player/porvata/porvata.js

```diff
@@ -24,6 +23,12 @@ define('ext.wikia.adEngine.video.player.porvata', [
 			return params.floatingContext && params.floatingContext.isActive();
 		}
 
+		function trackViewportConflict(conflictingAdSlot) {
+			params.conflictingAdSlot = conflictingAdSlot;
+			tracker.track(params, 'viewport-conflict');
+			params.conflictingAdSlot = undefined;
```

Comment:
I wish we were just sending a copy of `params` to `tracker.track` 😭

Reply:
It's because we don't have `Object.assign()` in ES5 :( But I don't like what I did anymore - refactored.